site/kube-addon.md

```diff
@@ -178,6 +179,73 @@ definition](http://kubernetes.io/docs/api-reference/extensions/v1beta1/definitio
   all multicast traffic) by adding `--allow-mcast` as an argument to
   `weave-npc` in the YAML configuration.
 
+## <a name="operations"></a><a name="troubleshooting"></a> Troubleshooting
+
+You can run `weave` CLI commands to confirm Weave Net' status. This can be done in various ways:
+
```

Comment:
The status of Weave is checked by running its CLI commands.   This can be done in various ways: